paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -40,14 +40,14 @@ class PaparazziPlugin : Plugin<Project> {
     val variants = project.extensions.getByType(LibraryExtension::class.java)
         .libraryVariants
     variants.all { variant ->
+      val paparazziResourcesDetailsFile = "intermediates/paparazzi/resources.txt"
```

Comment:
this will become the only usage, with the below suggestion, so this can be inlined.